front/lib/utils/router.ts

```diff
@@ -3,10 +3,24 @@ import type { NextRouter } from "next/router";
 export const setQueryParam = (
   router: NextRouter,
   key: string,
-  value: string
+  value: string,
```

Comment:
I guess we don't this update anymore, right?

Reply:
Not really, will rollback